src/AppBundle/Validator/ValidAddressValidator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use AppBundle\Address\AddressInterface;
+use AppBundle\Intl\FranceCitiesBundle;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * @Annotation
+ */
+class ValidAddressValidator extends ConstraintValidator
```

Comment:
I prefer to have `ValidAddress` and `GeocodableAddress` constraints and to group constraints with validators. What do you propose?